src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/ConstructorInitializerSymbolReferenceFinder.cs

```diff
@@ -76,22 +79,78 @@ protected override bool CanFind(IMethodSymbol symbol)
                 {
                     return true;
                 }
+                else if (methodSymbol.Parameters.IsDefaultOrEmpty
+                    && syntaxFactsService.IsConstructorToken(t))
```

Comment:
Why only if the parameters list is empty?  What about something like this:

```c#
class Base { 
     Base() { }
}

class Derived {
    Derived(int i) { }
}
```
```Base()``` is called from ```Derived(i)```